index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width">
+        <title>Weather report</title>
+        <link href="styles/index.css" rel="stylesheet" type="text/css" />
+        <script src="scripts/index.js"></script>
+    </head>
+    <body>
+        <nav>
+            <div class="row">
+                <div class="col-3 nav-col">
+                    <select class="form-rounded-corners" id="sky">
+                        <option value="sun">Sunny</option>
+                        <option value="cloudy">Cloudy</option>
+                        <option value="rainy">Rainy</option>
+                        <option value="snow">Snowy</option>
+                    </select>
+                </div>
+                <div class="col-3 nav-col">
+                    <input class="form-rounded-corners" type="text" id="city" placeholder="City">
+                </div>
+                <div class="col-4 nav-col">
+                    <div class="row">
+                        <div class="col-4 nav-col">
+                            <button class="tempSelector form-rounded-button" id="reduceTemp">-</button>
+                        </div>
+                        <div class="col-4 nav-col">
+                            <input class="form-rounded-corners" type="text" id="temperature_small">
+                        </div>
+                        <div class="col-4 nav-col">
+                            <button class="tempSelector form-rounded-button" id="increaseTemp">+</button>
+                        </div>                                                
+                    </div>
+                </div>    
+                <div class="col-2 nav-col">
+                    <button class="form-rounded-button" id="resetEverything">Reset</button>
+                </div>                                            
+            </div>
+        </nav>
+        <main id="main">
+            
+            <h1 class="temperature theMainText" id="temperature_big"></h1>
+            <div class="row">
+                <div class="col-12 landscape-uber-holder">
+                    <div class="landscape-city-holder" id="landscape-city-holder">
+                        <h2 class="theMainText" id="cityname"></h2>
+                        <img id="landscape-image" src="./assets/landscapes/fall.png"></div>
```

Comment:
For accessibility you should have an `alt` attribute.

You should also remove the `</div>` because you have 3 close divs below.